source/platforms/brev-dev.md

```diff
@@ -0,0 +1,82 @@
+## Section 1: Setting up your instance
+
+Navigate to the Brev.dev console and click on "Create your first instance"
+
+![Alt Text](/_static/images/platforms/brev/brev1.png)
+
+Select Container Mode
+
+![Alt Text](/_static/images/platforms/brev/brev2.png)
+
+Attach NVIDIA RAPIDS Container
+
+![Alt Text](/_static/images/platforms/brev/brev3.png)
+
+Configure your own instance
+
+![Alt Text](/_static/images/platforms/brev/brev4.png)
+
+And hit Deploy
+
+![Alt Text](/_static/images/platforms/brev/brev5.png)
+
+## Section 2: Accessing your instance
+
+To simply create and use a Jupyter Notebook:
+Click Open Notebook at the top right after the page has deployed
+
+On VS Code:  
+
+Go to VS Code and type:  
+```bash
+brev open
+```
+
+And it will automatically open a new VS Code window for you to use with RAPIDS. 
+
+On Terminal 
+Type: 
+
+```bash
+brev shell 
+```
+
+SSH
+**Forwarding a Port Locally:**
```

Comment:
Can you use markdown headings rather than bolded text. Sphinx uses them to generate the table of contents correctly.

```suggestion
### Forwarding a Port Locally
```